### PR TITLE
create Chinese version manual and  help

### PR DESCRIPTION
**Pull request by John0079**
the first time create Chinese version manual and help

### CODE REVIEW

**Comment by nicolas-raoul** [comment]
Hello John,

Thank you for starting this!

We will publish it when the translation is complete. It is a lot of work but good luck :-)
Thanks to GitHub, other contributors may help you by translating some parts of the text, so don't hesitate to point others to your branch.

Cheers!
Nicolas

**Comment by John0079** [comment]
Hi Nicolas,
I am very pleasure to work together with anki volunteers from around the world. I hope I can spend more time to complete translation as soon as possible!

Cheers!
John

**Comment by John0079** [comment]
help-zh-CN translation has completed, would you like to merge it into the document?

**Comment by timrae** [comment]
@John0079 

Hi John,

I think it will be easier to merge the help and manual files together, so I'll wait for some progress on the manual.

**Comment by timrae** [comment]
Good work by the way! :)

File: help-zh-CN.asc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+== 提问之前
+
+请求帮助之前, 请先试着先通过这个链接link:manual.html[AnkiDroid 手册]搜索类似问题,你也可以通过  https://github.com/ankidroid/Anki-Android/wiki/FAQ[常见问题解答]列表, 以及官方 link:http://ankisrs.net/docs/manual.html[Anki手册] 查找常见的Anki系统问题，这里的阐述的anki的特点不仅限于android，它适用于各种anki软件系统.
+
+== 支持
+如果你不能从上面提供的文档中找到你所需要的，请进入下面相关网站：
+
+非android特定问题 :: AnkiDroid (Anki的android版本) 不同于其他版本，被单独创建和管理的. Android版本不负责PC / Web / iOS 版本, 以及一般的 Anki 问题, 这些版本遇到的问题请参阅 https://anki.tenderapp.com/[Anki 支持网站].
+
+AnkiDroid问题 :: 有关AnkiDroid的帮助和支持, 请参阅 https://groups.google.com/forum/#!forum/anki-android[用户论坛], 或者你你不想再论坛上注册，你也可以将你的问题发送到这个论坛的 mailto:public-forum@ankidroid.org[公共邮箱] .
```
Comment (timrae) [inline help-zh-CN.asc]: @John0079 
I think you'll need to edit this part a little bit. We get a lot of spam spent to the main forum in Chinese and Korean, so there's a very high chance that we delete messages in these languages. I suggest noting that the forum and email address above are English only, and to give a link to your Chinese forum for users that cannot write in English.

Comment (John0079) [inline help-zh-CN.asc]: @timrae 
good idea, Ankichina forum has just created, it can provide a communication space for the user to discuss the problem.

**Comment by timrae** [comment]
@John0079 

Thanks for pushing the new commits! I noticed that some of the hyperlinks are not working properly:

https://github.com/ankidroid/ankidroiddocs/blob/38c766310784b60e0728a73aed4c44372a1aa995/manual-zh-CN.asc#回答特征的类型

Could you please fix them?

**Comment by John0079** [comment]
yes , I found these problems, I will try to fix them.

**Comment by John0079** [comment]
thanks, it is great! let's cheers

**Comment by timrae** [comment]
Thank you for the good work :)

**Comment by John0079** [comment]
I am glad to do it, it's my pleasure.

**Comment by agrueneberg** [comment]
This is great, John! Thank you so much for this and the work you have put into ankichina.net—I don't speak Chinese, but it looks like a useful resource.

**Comment by John0079** [comment]
Thanks, i am glad to do it, then , I will do more things about anki, i want to go ahead with this team!

**Pull request merged**